site/src/@ndlib/gatsby-theme-marble/components/Pages/BrowsePage/index.js

```diff
@@ -50,36 +50,36 @@ const Browse = ({ location }) => {
   const formats = [
     {
       label: 'Paintings',
-      target: '/search?format[0]=Paintings',
+      target: '/search?format[0]=paintings',
     },
     {
       label: 'Maps',
       target: '/search?format[0]=Cartographic%20material',
     },
     {
       label: 'Photographs',
-      target: '/search?format[0]=Photographs',
+      target: '/search?format[0]=photographs',
     },
     {
       label: 'Prints and Posters',
-      target: '/search?format[0]=Prints&format[1]=Two-dimensional%20nonprojected%20graphic&format[2]=Projected%20medium',
+      target: '/search?format[0]=prints&format[1]=Two-dimensional%20nonprojected%20graphic&format[2]=Projected%20medium',
```

Comment:
You missed some capitals in the 2nd and 3rd query parameters here, if the intend was to use all lowercase.